local-volume/examples/local-reader.yaml

```diff
@@ -24,4 +24,4 @@ spec:
       volumes:
       - name: local-vol
         persistentVolumeClaim: 
-          claimName: "local-vol-local-test-0"
+          claimName: "example-local-claim"
```

Comment:
This local reader example is paired with the statefulset example, not manual pv.  It has a bash script that reads the contents of the statefulset.  The simple pvc example is for manual pv.